src/mmath_bin.erl

```diff
@@ -68,3 +71,6 @@ realize(<<>>, _, Acc) ->
 
 derealize(Data) ->
     << <<?INT:?TYPE_SIZE, D:?BITS/?INT_TYPE>> || <<D:64/integer-native>> <= Data>>.
```

Comment:
Line 73 is too long:     << <<?INT:?TYPE_SIZE, D:?BITS/?INT_TYPE>> || <<D:64/integer-native>> <= Data>>..